packages/backend/test/exploregens.spec.ts

```diff
@@ -172,42 +231,57 @@ describe("exploregens unit test", () => {
         .resolves();
       windowMock
         .expects("setStatusBarMessage")
-        .withExactArgs(messages.installing(genName))
-        .returns({ dispose: () => "" });
+        .withArgs(messages.installing(genName))
+        .returns({
+          dispose: () => {
+            return;
+          },
+        });
 
       await exploregens["install"](gen);
     });
   });
 
   describe("getFilteredGenerators", () => {
```

Comment:
please consider to add a negative tests

Reply:
for each uninstall, install and update methods there are 2 types of tests: positive and negative
One flow passes successfully and another one fails.